lib/cmsis_rtos_v1/cmsis_msgq.c

```diff
@@ -78,6 +80,7 @@ osEvent osMessageGet(osMessageQId queue_id, uint32_t millisec)
 
 	if (retval == 0) {
 		evt.status = osEventMessage;
+		evt.value.v = info;
```

Comment:
What you did is ok. However, please make evt 0 by default. i.e the statement "osEvent evt;" should be changed to "osEvent evt = {0};" at the top.
Also, this change is applicable for mailbox too. Please do the same modification there as well and push as part of the same PR.